config/templates/managed-velero-operator-csv-template.yaml

```diff
@@ -0,0 +1,125 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: managed-velero-operator.v0.0.1
+  namespace: placeholder
+  annotations:
+    certified: "false"
+    description: Operator to manage installation of Velero in OpenShift.
+    containerImage: quay.io/app-sre/managed-velero-operator:latest
+    createdAt: "2019-11-11T11:00:00Z"
+    support: Christoph Blecker
+    alm-examples: '[{"apiVersion":"managed.openshift.io/v1alpha1","kind":"Velero","metadata":{"name":"cluster","namespace":"openshift-velero"}}]'
+spec:
+  displayName: Managed Velero Operator
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: Velero
+      name: veleros.managed.openshift.io
+      version: v1alpha1
+  description: Operator to manage installation of Velero in OpenShift.
```

Comment:
I just noticed, this indentation is incorrect see: https://gitlab.cee.redhat.com/service/saas-aws-account-operator-bundle/blob/staging/aws-account-operator/0.1.438-96d664f/aws-account-operator.v0.1.438-96d664f.clusterserviceversion.yaml#L16-20